### PR TITLE
[DOC] Update `mr_proc` Overview page

### PR DESCRIPTION
**Pull request by michellewang**
Changes proposed in this pull request:
- Update Overview page following discussion with @nikhil153. Notable changes include the addition of a "Who is `mr_proc` for?" section featuring 2 user profiles (data managers and data users), as well as an FAQ with some anticipated questions
- Add minor formatting changes to some of the other files

## Checklist

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[BUG]`, `[DOC]`, `[INFRA]`, `[MAINT]`)
- [ ] PR links to Github issue with mention `Closes #XXXX`
- [x] Tests pass
- [ ] Code is properly formatted

### CODE REVIEW

**Comment by alyssadai** [comment]
Hey @michellewang and @nikhil153, this is looking great.

Since we may have more eyes on the documentation now as some folks from Alain's lab look into using various modules, any chance we could merge this soon?

Also, could we replace `mr_proc` with `nipoppy` everywhere on this Overview page?

**Comment by alyssadai** [comment]
Before I forget, I think it would also be helpful to have a link to the [nipoppy repo](https://github.com/neurodatascience/nipoppy) somewhere on the overview page. Wdyt?

**Comment by michellewang** [comment]
@nikhil153 I changed most references to `mr_proc` to Nipoppy, let me know if this looks okay. Only exception should be `mr_proc_manifest.csv` since we haven't changed the codebase for that yet.

File: docs/nipoppy/installation.md [hunk 1]
```diff
@@ -2,28 +2,29 @@
...
+
+### Nipoppy dataset directory setup 
+
+Run `tree.py` to create the Nipoppy dataset directory tree:
 ```bash
 python tree.py --mr_proc_root <DATASET_ROOT>
 ```
-
+Where
 - `data_disk`: data storage location on a local disk
```
Comment (nikhil153) [inline docs/nipoppy/installation.md:26]: `data_dist` is deprecated

File: docs/nipoppy/overview.md [hunk 2]
```diff
@@ -1,39 +1,90 @@
 ## What is Nipoppy (formerly mr_proc)? 
-*Process long and prosper*
 
----
+[*Process long and prosper*](https://en.wikipedia.org/wiki/Vulcan_salute)
```
Comment (nikhil153) [inline docs/nipoppy/overview.md:3]: I think we can remove this now :)

File: docs/nipoppy/overview.md [hunk 3]
```diff
@@ -1,39 +1,90 @@
...
----
+### Example workflow
 
-## Modules
+Given a dataset to process, a typical workflow with Nipoppy can look like this:
 
-1. `Code`: Codebase [repo](code_org.md) for running and tracking workflows
-2. `Data`: [Dataset](data_org.md) organized in a specific directory structure
-3. `Containers`: Singularity containers encapsulating your processing pipelines
+1. Clone the [Nipoppy template code repository](https://github.com/neurodatascience/nipoppy)
```
Comment (nikhil153) [inline docs/nipoppy/overview.md:16]: Do we want people to clone directly or first fork it for their dataset?

File: docs/nipoppy/overview.md [hunk 4]
```diff
@@ -1,39 +1,90 @@
...
-1. Standardized data i.e. convert DICOMs into BIDS
-2. Run commonly used image processing pipelines e.g. FreeSurfer, fMRIPrep
-3. Organize processed MR data inside `derivatives` directory
-4. Organize demographic and clinical assessment data inside `tabular` directory
-5. Run tracker scripts to populate `bagel.csv` with tabular and processing pipeline metadata
-6. Provide metadata to `NeuroBagel` to allow dashboarding and querying participants across multiple studies
-    
+* People who use tabular, derivative or other files produced by Nipoppy (e.g., from a shared Nipoppy-compliant dataset)
+* Example use cases:
+    * Data tracking
```
Comment (nikhil153) [inline docs/nipoppy/overview.md:51]: Date querying would be better phrasing for "data users"

File: docs/nipoppy/overview.md [hunk 5]
```diff
@@ -1,39 +1,90 @@
...
+![nipoppy_org](../imgs/nipoppy_org.jpg)
+
+### Steps
+The Nipoppy workflow steps and linked identifiers (i.e. `participant_id`, `dicom_id`, `bids_id`) are shown below:
 
+![steps](../imgs/steps.jpg)
 
-## Steps
-The Nipoppy workflow steps and linked identifiers (i.e. participant_id, dicom_id, bids_id) are show below:
+## FAQ
```
Comment (nikhil153) [inline docs/nipoppy/overview.md:77]: Question /  Answers to add
- Do I need to install anything?
- Can I use Nipoppy on a cluster?
- How do I use the dashboard?

File: docs/nipoppy/workflow/proc_pipe/fmriprep.md [hunk 6]
```diff
@@ -44,7 +44,7 @@ python run_fmriprep.py \
 
 !!! note
 
-    Unlike DICOM and BIDS run scripts, `run_fmriprep.py` can only process 1 participant at a time due to heavy compute requirements of fMRIPrep. For parallel processing on a cluster, sample HPC job scripts (slurm and sge) are provided in [hpc](https://github.com/neurodatascience/mr_proc/tree/main/workflow/proc_pipe/fmriprep/scripts/hpc) subdir. 
+    Unlike DICOM and BIDS run scripts, `run_fmriprep.py` can only process 1 participant at a time due to heavy compute requirements of fMRIPrep. For parallel processing on a cluster, sample HPC job scripts (slurm and sge) are provided in [hpc](https://github.com/neurodatascience/nipoppy/tree/main/workflow/proc_pipe/fmriprep/scripts/hpc) subdir.
```
Comment (nikhil153) [inline docs/nipoppy/workflow/proc_pipe/fmriprep.md:47]: The path to hpc scripts is updated:
https://github.com/neurodatascience/nipoppy/tree/main/workflow/proc_pipe/fmriprep/scripts

**Review by nikhil153 — COMMENTED** [review]
Overall looks good! Mostly minor changes needed before the merge.

File: docs/nipoppy/overview.md (hunk 2 above)
Comment (michellewang) [reply docs/nipoppy/overview.md:3]: Some people said they liked it, and I thought it was neat even though I didn't know it was from Star Trek. So I think we can keep it for now

File: docs/nipoppy/overview.md (hunk 3 above)
Comment (michellewang) [reply docs/nipoppy/overview.md:16]: Fork, updated

**Pull request merged**